### ISSUE TITLE
API Integration

### ISSUE THREAD

**Issue by freakNewton** [BosamiyaSimran/SE-Project-Group-15#10]
@BosamiyaSimran how will we do the integration of the API once we get the selected text to the backend??

**Comment by BosamiyaSimran** [comment]
Have started working on using googleapi for translating selected text. Facing issue with google-translate api versions (there are two versions v2 and v3). Can you please help me with fixing it? @freakNewton 
 Also, earlier we were planning to use JS/jQuery directly but  let us use node.js and express.js as they are more flexible.

**Comment by nsshah14** [comment]
Yes we can start with node.js. We might get more references to complete our project.
